src/test/utils/expressMocks/Response.ts

```diff
@@ -0,0 +1,23 @@
+import { EventEmitter } from 'events';
+import * as t from 'jest-express';
+
+// jest-express.Response does not implement EventEmitter
+// so piping to the response does not work
+export default class Response extends t.Response implements EventEmitter {
+  public on() { return this; }
+  public write() { return this; }
+  public once() { return this; }
+  // public addListener() { return this; }
```

Comment:
Aren't these methods that you commented required?

Reply:
Coverage Said they weren’t used so no

Reply:
They're mostly there to fulfill EventEmitter interface. Haven't checked out the code, but I suppose tslint will complain.